dart/lib/src/sentry_client.dart

```diff
@@ -109,14 +101,18 @@ class SentryClient {
       platform: event.platform ?? sdkPlatform,
     );
 
-    if (event.throwable != null && event.exception == null) {
+    if (stackTrace != null && event.throwable == null) {
```

Comment:
you should test `event.exception == null` here too

Reply:
✅